README.asciidoc

```diff
@@ -49,17 +49,17 @@ Before you start running the unit and integration tests in the Neo4j Maven proje
 
 == Running Neo4j ==
 
-After running a `mvn clean install` cd into `packaging/standalone/target` and extract the version you want, then
+After running a `mvn clean install`, **cd** into `packaging/standalone/target` and extract the version you want, then:
```

Comment:
We use the code syntax for actual commands.
```suggestion
After running a `mvn clean install`, `cd` into `packaging/standalone/target` and extract the version you want, then:
```

Reply:
changes applied to the underlying commit. Thanks.